README.md

```diff
@@ -207,6 +239,7 @@ for more detailed instruction on app definitions.
 |-----------------|---|
 |2.0|optional|
 |3.0|optional **[Deprecated]**|
```

Comment:
This was a deprecation warning. Command.json is still supported in 3.0. It's just optional (users don't have to specify it). Maybe we should remove it the warning given that we don't support it for 4.0. I don't know @jsancio What do you think?